inventory/hosts.example

```diff
@@ -637,6 +637,36 @@ debug_level=2
 #openshift_persistentlocalstorage_path=/mnt/local-storage
 #openshift_persistentlocalstorage_provisionner_image=quay.io/external_storage/local-volume-provisioner:v1.0.1
 
+# Cluster monitoring
+#
+# Cluster monitoring is enabled by default, disable it by setting
+# openshift_cluster_monitoring_operator_install=false
+#
+# Cluster monitoring configuration variables allow setting the amount of
+# storage requested through PersistentVolumeClaims, which node-selector to use
+# to run components of cluster-monitoring on and the configuration to use for
+# the Prometheus Alertmanager.
+#
+# openshift_cluster_monitoring_operator_prometheus_storage_capacity: "50Gi"
```

Comment:
These should be in INI format, so `openshift_cluster_monitoring_operator_prometheus_storage_capacity="50Gi"`

The switch to YAML is planned, but we'll take care of it